_data/simple-icons.json

```
@@ -15567,6 +15567,11 @@
             "hex": "1477D1",
             "source": "https://www.tinkercad.com"
         },
+        {
+            "title": "Tinygrad",
```

```suggestion
            "title": "tinygrad",
```
They always refer themselves as `tinygrad` with a lowercase `t`.